e2e/app/component/modal.ts

```diff
@@ -32,13 +32,17 @@ export default class Modal extends Container {
     return this;
   }
 
-  async getContent(): Promise<string> {
+  async getTextContent(): Promise<string[]> {
```

Comment:
I like `getTextContent` better.
Function used to returns all strings as one long string of texts. It was not very useful for test assertions.
Function now returns string array. It can be used to verify modal title and text messages independently.